app/src/main/java/com/example/navadroid/androidonclickevent/Option1Activity.java

```diff
@@ -36,11 +37,25 @@ public void process(View v) {
         hideKeyboardInput(v);
     }
 
+    public void Next1(View nextButton1) {
```

Comment:
Same as Option0, you may merge these methods into "process" and use if-else to select which view is the sender.